torba/torba/workbench/_blockchain_dock.py

```diff
@@ -1,5 +1,3 @@
-# -*- coding: utf-8 -*-
-
 # Form implementation generated from reading ui file 'blockchain_dock.ui',
 # licensing of 'blockchain_dock.ui' applies.
```

Comment:
This file is generated and should not be edited.

Reply:
Reverted!